src/Forms/Prism.Forms/Services/Dialogs/IDialogResult.cs

```diff
@@ -2,9 +2,20 @@
 
 namespace Prism.Services.Dialogs
 {
+    /// <summary>
+    /// Contains <see cref="IDialogParameters"/> from the dialog
```

Comment:
no need to duplicate the properties of the interface. Just briefly explain what the interface is used for. 

Something like:
Represents a class that returns information about a dialog that has been closed.

Reply:
I refer to your advice.  
[https://github.com/PrismLibrary/Prism/pull/2761/files#diff-627f40b52b5622df540ef8100476c49cc13a491a5c1e17177ce2ab9a409d0e97R6](https://github.com/PrismLibrary/Prism/pull/2761/files#diff-627f40b52b5622df540ef8100476c49cc13a491a5c1e17177ce2ab9a409d0e97R6)

But, in the implementation class, I change the wording slightly.  
https://github.com/PrismLibrary/Prism/pull/2761/files#diff-0d0a9b1f968cdc60f0420e45398c06795f2b4d473269f729f6a2f977c96ac071R9